docs/core/testing/unit-testing-mstest-writing-tests-attributes.md

```diff
@@ -201,23 +201,7 @@ public class MyTestClass
     }
 
     [AssemblyCleanup]
-    public static void AssemblyCleanup()
-    {
-    }
-}
-```
-
-```csharp
-[TestClass]
-public class MyOtherTestClass
```

Comment:
I didn't see the point of this "MyOtherTestClass". AssemblyInitialize/Cleanup should be defined only once.

Reply:
Oh I didn't notice one was void and the other it Task. I'll add that back